src/Sarif.Viewer.VisualStudio.ResultSources.GitHubAdvancedSecurity.Core/Services/GitHubSourceService.cs

```diff
@@ -188,7 +188,7 @@ public async Task InitializeAsync()
         /// <inheritdoc cref="IGitHubSourceService.IsActiveAsync()"/>
         public async Task<Result> IsActiveAsync()
         {
-            if (this.GetOptionStateCallback != null && this.GetOptionStateCallback("GitHubAdvancedSecurity"))
+            if (this.GetOptionStateCallback != null && (bool)this.GetOptionStateCallback("GitHubAdvancedSecurity"))
```

Comment:
>(bool) [](http://example.com/codeflow?start=55&length=6)

I'd prefer TryParse just in case something is wonky with the menu item

Reply:
You can call Object.ToString() which will be "True" or "False" if it's a boolean